### DISCUSSION TITLE
Installing Breeze doesn't automatically add APP_FRONTEND_URL to the .env file

### DISCUSSION THREAD

**Discussion by 0xgdr** [laravel/framework#39932] [General]
I am not sure if this is a an issue with the Breeze package or not. The docs state:

> During installation, Breeze will add an APP_FRONTEND_URL environment variable to your application's .env file.

https://laravel.com/docs/8.x/starter-kits#breeze-and-next

Laravel version 8.75.0

When I install Laravel using Sail and Breeze using `sail composer require laravel/breeze --dev` the `APP_FRONTEND_URL` variable is not automatically added to my .env file. Not a big issue at all because it's easy to add but thought I would mention it.

**Comment by ghost** [comment]
Seeing this also after a fresh install of breeze using sail. Is it an issue with the wording in the docs? or a bug?
